doc/user/radial_and_axial_expansion.rst

```diff
@@ -0,0 +1,86 @@
+*******************************************
+Radial and Axial Expansion and Contraction
+*******************************************
+
+ARMI natively supports both one-dimensional radial and axial expansion. These expansion types function independently of one another and each have their own set of underlying assumptions and use-cases. The remainder of this section is described as follows: in Section :ref:`thermalExpansion` the methodology used for thermal expansion within ARMI is described; in Sections :ref:`radialExpansion` and :ref:`axialExpansion`, we describe the design, limitations, and intended functionality of radial and axial expansion, respecitely.
+
+.. _thermalExpansion:
+
+Thermal Expansion
+-----------------
+ARMI treats thermal expansion as a linear phenomena using the standard linear expansion relationship,
+
+.. math::
+    \frac{\Delta L}{L_0} = \alpha(T) \Delta T,
+    :label: linearExp
+
+where, :math:`\Delta L` and :math:`\Delta T` are the change in length and temperature from the reference state, respectively, and :math:`\alpha` is the thermal expansion coefficient relative to :math:`T_0`. Expanding Equation :eq:`linearExp` we obtain,
+
+.. math::
+    \frac{L_1 - L_0}{L_0} = \alpha(T_1)\left(T_1 - T_0\right).
+    :label: linearExp2
+
+Rearranging Equation :eq:`linearExp2`, we can obtain an expression for the new length, :math:`L_1`,
```

Comment:
I think you could probably just skip directly to this equation.

Reply:
Up to you.

Reply:
https://github.com/terrapower/armi/pull/1204/commits/656cffbcf8170112b6d8f96d0b2c710d73115ef1